### PR TITLE
Feature/meshery updates

### PR DESCRIPTION
**Pull request by girishranganathan**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -14,4 +14,4 @@ setup:
 	ADMIN=on bundle install
 
 docker:
-	docker run --rm -p 4000:4000 -v `pwd`:"/srv/jekyll" jekyll/jekyll:3.7.3 bash -c "bundle install; jekyll serve --drafts"
+	docker run --name layer5 --rm -p 4000:4000 -v `pwd`:"/srv/jekyll" jekyll/jekyll:3.7.3 bash -c "bundle install; jekyll serve --drafts"
```
Comment (venilnoronha) [inline Makefile]: Should this be `meshery` instead?

File: _includes/meshery.html [hunk 2]
```diff
@@ -52,19 +52,161 @@ <h5 style="color:#111111;">Problem</h5>
...
+            <li style="list-style-type: circle;">accepts and maintains the Kubernetes cluster config and context info for the period of the session</li>
+            <li style="list-style-type: circle;">enables users to run load tests against their applications</li>
+            <li style="list-style-type: circle;">collects the results of the load tests from the load generator and passes it to Meshery SaaS</li>
+            <li style="list-style-type: circle;">interfaces with the Meshery adapters dynamically and enables the user to play with the service mesh</li>
+          </ol>
+        </p>
+      <li>Meshery Adapters</li>
+        <p>
+          Meshery adapters are the components which will be interfacing with the specific service meshes to apply the config changes requested through Meshery App. 
+          Meshery app will be interfacing with the relevant adapters over gRPC. The adapters will be running the gRPC server and the Meshery App will spin up the gRPC client as and when an operation is requeted through the user interface.
```
Comment (venilnoronha) [inline _includes/meshery.html]: s/requeted/requested

File: _includes/meshery.html [hunk 3]
```diff
@@ -52,19 +52,161 @@ <h5 style="color:#111111;">Problem</h5>
...
+          The proto buf spec is kept in the Meshery App so as to keep it in a central location. 
+          Today we have a functional adapter for Istio which is named <b>meshery-istio</b>. 
+          The adapters will be maintained in their own git repository. Source code for <b>meshery-istio</b> can be found <a href="https://github.com/layer5io/meshery-istio">here</a>.
+          We have also created a repository <a href="https://github.com/layer5io/meshery-linkerd">here</a> for working with <a href="https://linkerd.io/2/getting-started/">Linkerd2</a>.
```
Comment (venilnoronha) [inline _includes/meshery.html]: s/Linkerd2/Linkerd 2.0

File: _includes/meshery.html [hunk 4]
```diff
@@ -52,19 +52,161 @@ <h5 style="color:#111111;">Problem</h5>
...
+          </li>
+        </ol>
+      </li>
+      <li style="list-style-type: circle;">On Docker
+        <ol>
+          <li>We have a docker-compose.yaml file which can be used to spin up the services quickly</li>
+          <li>There a few requirements for running all the Meshery services on your local
+            <ol>
+              <li>
+                <p>SSO, which uses Twitter and/or Github</p>
```
Comment (venilnoronha) [inline _includes/meshery.html]: s/Github/GitHub

File: _includes/meshery.html [hunk 5]
```diff
@@ -52,19 +52,161 @@ <h5 style="color:#111111;">Problem</h5>
...
+                <ol>
+                <li style="list-style-type: circle;">Instructions to setup Twitter for SSO can be found <a href="#using-twitter-for-sso">below</a></li>
+                <li style="list-style-type: circle;">Instructions to setup Github for SSO can be found <a href="#using-github-for-sso">below</a></li>
```
Comment (venilnoronha) [inline _includes/meshery.html]: s/Github/GitHub

File: _includes/meshery.html [hunk 6]
```diff
@@ -52,19 +52,161 @@ <h5 style="color:#111111;">Problem</h5>
...
+                <div class="highlight highlight-source-shell">
+                <pre><code>TWITTERKEY="PASTE TWITTER KEY"
+                </code></pre>
+                </div>
+                <div class="highlight highlight-source-shell">
+                <pre><code>TWITTERSECRET="PASTE TWITTER SECRET"
+                </code></pre>
+                </div>
+                <ol>
+                  <li style="list-style-type: circle;">for Github:</li>
```
Comment (venilnoronha) [inline _includes/meshery.html]: s/Github/GitHub

File: _includes/meshery.html [hunk 7]
```diff
@@ -52,19 +52,161 @@ <h5 style="color:#111111;">Problem</h5>
...
+                </ol>
+                <div class="highlight highlight-source-shell">
+                <pre><code>GITHUBKEY="PASTE GITHUB KEY"
+                </code></pre>
+                </div>
+                <div class="highlight highlight-source-shell">
+                <pre><code>GITHUBSECRET="PASTE GITHUB SECRET"
+                </code></pre>
+                </div>
+                <p><strong>Note</strong>: you can use Twitter and/or Github</p>
```
Comment (venilnoronha) [inline _includes/meshery.html]: s/Github/GitHub

File: _includes/meshery.html [hunk 8]
```diff
@@ -52,19 +52,161 @@ <h5 style="color:#111111;">Problem</h5>
...
+      <li>Create an app in the Twitter developer console: <a href="https://developer.twitter.com/en/apps" rel="nofollow">https://developer.twitter.com/en/apps</a> after logging in.</li>
+      <li>Fill appropriate details in the presented form
+      <ol>
+      <li style="list-style-type: circle;">Remember to enable to <code>Sign in with Twitter</code></li>
+      <li style="list-style-type: circle;">For the callback url, please use this value: <code>http://meshery-saas:9876/auth/twitter/callback</code></li>
+      </ol>
+      </li>
+      <li>After creating the app you will be able to grab the API key and secret from the <code>Keys and tokens</code> section of the app.</li>
+      </ol>
+      <h6><a name="using-github-for-sso">Using Github for SSO</a></h6>
```
Comment (venilnoronha) [inline _includes/meshery.html]: s/Github/GitHub

File: _includes/meshery.html [hunk 9]
```diff
@@ -52,19 +52,161 @@ <h5 style="color:#111111;">Problem</h5>
...
+      <p>
+        <ol>
+        <li>Create an OAuth app in the Github developer settings: <a href="https://github.com/settings/developers">https://github.com/settings/developers</a> after logging in.</li>
```
Comment (venilnoronha) [inline _includes/meshery.html]: s/Github/GitHub

File: Makefile (hunk 1 above)
Comment (girishranganathan) [reply Makefile]: Good question. This docker command is for running the layer5 site, hence layer5 was my thought.

File: _includes/meshery.html (hunk 2 above)
Comment (girishranganathan) [reply _includes/meshery.html]: great, will get this fixed

File: Makefile (hunk 1 above)
Comment (venilnoronha) [reply Makefile]: Oops, I just realized that this was running `jekyll`.

**Pull request merged**